src/shared/elf-util.c

```diff
@@ -430,11 +431,9 @@ static int module_callback(Dwfl_Module *mod, void **userdata, const char *name,
         Elf *elf;
 
         assert(mod);
+        assert(name);
```

Comment:
this is a callback, and `name` is not guaranteed to be set by libdwfl as far as I understand